cupy/core/_scalar.pyx

```diff
@@ -0,0 +1,220 @@
+import numpy
+import six
+
+from cupy.core import _dtype
+from cupy.core cimport internal
+
+
+cdef dict _typenames_base = {
```

Comment:
Are these mapping safe?
e.g. can we assume `long long` == `int64`?

Reply:
OK. I do not change this code.